src/Categories/Bicategory/Monad/Bimodule.agda

```diff
@@ -0,0 +1,99 @@
+{-# OPTIONS --without-K --safe --lossy-unification #-}
+
+open import Categories.Bicategory
+
+module Categories.Bicategory.Monad.Bimodule {o ℓ e t} {𝒞 : Bicategory o ℓ e t} where
+
+open import Level
+open import Categories.Bicategory.Monad using (Monad)
+import Categories.Bicategory.Extras as Bicat
+open Bicat 𝒞
+
+record Bimodule (M₁ M₂ : Monad 𝒞) : Set (o ⊔ ℓ ⊔ e) where
+  open Monad using (C; T; μ; η)
+  field
+    F       : C M₁ ⇒₁ C M₂
+    actionˡ : F ∘₁ T M₁ ⇒₂ F
+    actionʳ : T M₂ ∘₁ F ⇒₂ F
+
+    assoc     : actionʳ ∘ᵥ (T M₂ ▷ actionˡ) ∘ᵥ associator.from ≈ actionˡ ∘ᵥ (actionʳ ◁ T M₁)
+    sym-assoc : actionˡ ∘ᵥ (actionʳ ◁ T M₁) ∘ᵥ associator.to ≈ actionʳ ∘ᵥ (T M₂ ▷ actionˡ)
```

Comment:
Since you use this version that is "definitionally symmetric", it might be useful to have a `BimoduleHelper` that lets users elide the items that are derivable.